src/GitVersion.Core/VersionCalculation/VersionSearchStrategies/ConfiguredNextVersionVersionStrategy.cs

```diff
@@ -8,19 +8,45 @@ namespace GitVersion.VersionCalculation;
 /// BaseVersionSource is null.
 /// Does not increment.
 /// </summary>
-internal class ConfiguredNextVersionVersionStrategy(Lazy<GitVersionContext> versionContext) : VersionStrategyBase(versionContext)
+internal sealed class ConfiguredNextVersionVersionStrategy(Lazy<GitVersionContext> contextLazy) : IVersionStrategy
```

Comment:
Hmm. The reason behind this is because we are using a convention. If you see the enum VersionStrategies you will find one value with the name `ConfiguredNextVersion`. All values in this enum have a dedicated VersionStrategy implementation... Concatinating the enum value with VersionStrategy. Should I rename all VersionStrategy classes then?